### ISSUE TITLE
Sprint 43 (3/28/24-4/10/24)

### ISSUE THREAD

**Issue by lmorris3** [department-of-veterans-affairs/va.gov-team#79494]
## PTO
Kirk: 3/28-3/29
Steve: 3/28-3/29
Mateo: 4/8

## Sprint 43 Goals

**Campaigns**
_High-level goal: Prepare for behavior change communications campaign pause lifting._ 

Engineering:
- USiP PR, launch (blocked until SSiT OCTO gives word)

Design/Research:
- Deploy GovDelivery segmented emails (blocked, may unblock this sprint)

Data:
- Monitor sign in and call center data (blocked until campaigns resume)
- Automate MPI, SSOe, MHV data ingest
- finish source account creation data view
- pull monthly CSP usage numbers, send to Tom 

**Stretch Goals**

**IPP**
_High level goal: Prepare for IPP Service Pilot 1_

Engineering: Launch IPP Applications in LHDI DEV/QA regions.
- Start VASI scan, eMASS, WASA, PTA security countermeasures for LHDI
- Discovery - LHDI steps to deploy to DEV/QA
- Deploy IPP app to LHDI DEV/QA regions
- Get Datadog access for IPP logging and monitoring
- Configure IPP App with Datadog
- Intake DB integration - usercase model 
- Proofing Agent data model integration 
- Hide Data fields if null (map screen)
- Update ERD of IPP data model
- Start on Section 4 Development Countermeasures
- Start on Section 3 Architecture Countermeasures

Research/Design: 
- Pilot Site Executive leadership request letter
- Pilot mock email draft
- Note-taking tool finalization
- 4/1 presentation
- A11y updates - design (4 total)
- Complete Design for in-person appointment changes - Veteran Intake
- Complete Design Veteran language selection tool
- Remove acronyms from designs
- Veteran caretaker research - desktop

**Stretch Goals**
- Make research recommendations from PA UT insights
- Walk on onboarding flow
- Engineering A11y improvements

## What we want to demo at the end of the sprint

- LHDI deployment
- PA Research recs
- IPP appointment change designs
- Language selection
